wies/core/jinja2/colleague_detail.html

```diff
@@ -33,12 +33,25 @@
           {{ object.get_source_display() }}
         {% endif %}
       </dd>
-      <dt>Merk</dt>
+      <dt>Labels</dt>
       <dd>
-        {% if object.brand %}
-          {{ object.brand.name }}
+        
+        {% if object.user %}
+        {% set labels = object.user.labels.select_related('category').all() %}
```

Comment:
does this make sense to you ruben? that the labels on the userlist take precendence over the ones from the colleague record?

the reason these list are separate is that you can have collegues without user and users without colleague